
<launch>
  
  <arg name="odom_topic" default="odom_baselink" />
 
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find bada_2dnav)/param/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
    <rosparam file="$(find bada_2dnav)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find bada_2dnav)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find bada_2dnav)/param/global_costmap_params.yaml" command="load" /> 
    <rosparam file="$(find bada_2dnav)/param/dwa_local_planner_params.yaml" command="load" />
    <rosparam file="$(find bada_2dnav)/param/move_base_params.yaml" command="load" />
    <rosparam file="$(find bada_2dnav)/param/global_planner_params.yaml" command="load" />
    <rosparam file="$(find bada_2dnav)/param/navfn_global_planner_params.yaml" command="load" />


    <remap from="odom" to="$(arg odom_topic)"/>

  </node>
</launch>
